language/spec/language_samples_spec.rb

```diff
@@ -67,14 +101,14 @@ def capture &block
         project_id:   @project_id,
         storage_path: "gs://#{@bucket_name}/positive.txt"
       )
-    }.to output(/^\d\.\d+ \(\d\.\d+\)/).to_stdout
+    }.to output(/Sentence .*\n^.*\(\d\.\d+\)$\n^.*\(\d\.\d+\)$/).to_stdout
 
     expect {
       sentiment_from_cloud_storage_file(
         project_id:   @project_id,
         storage_path: "gs://#{@bucket_name}/negative.txt"
       )
-    }.to output(/^-\d\.\d+ \(\d\.\d+\)/).to_stdout
+    }.to output(/Sentence .*\n^.*\(-\d\.\d+\)$\n^.*\(-\d\.\d+\)$/).to_stdout
```

Comment:
This test should expect output to include information understood from `text_content`